.github/workflows/ci.yml

```
@@ -146,22 +156,23 @@ jobs:
             env | grep '^JAVA'
             touch emulator.log
             chmod 770 emulator.log
-            adb logcat >> emulator.log &
+            adb logcat -c
```

minor nit: I'd prefer long options here so it takes less time for folks unfamiliar with adb to grok what's going on:

```suggestion
            adb logcat --clear
```

On that note I was surprised we wanted to clear the log here, but the `--help` output says:

> Clear (flush) the entire log and exit.

I suppose it's the "flushing" behaviour we're interested in?

---

That's correct. I figured it was better to flush these ahead-of-time just in case we implement AVD caching in the future to speed CI up.